configuration/configuration.go

```diff
@@ -10,6 +10,43 @@ import (
 	"time"
 )
 
+// The HTTP struct represents the http section of the yaml
+// configuration file
+type HTTP struct {
```

Comment:
Sorry for the nit but `Configuration` should be the first declaration in this file.